roles/traefik/templates/traefik.toml

```diff
@@ -1,6 +1,6 @@
 [entryPoints]
   [entryPoints.web]
-    address = ":80"
+    address = ":traefik_port_http"
```

Comment:
This isn't an ansible variable, hence won't be templated

Reply:
You should take another look at:
https://github.com/davestephens/ansible-nas/blob/master/roles/traefik/defaults/main.yml